principles/devops.md

```diff
@@ -5,5 +5,52 @@ title:  "DevOps Principles"
+
+There are many blogs, books and courses on DevOps, and the CAMS model is often written about. The CAMS model was invented by Damon Edwards and John Willis, authors of the famous Podcast, DevOps Cafe. CAMS stands for Culture, Automation, Measurement and Sharing. Using the CAMS model, we can break down our core principles and provide an insight into how we deliver and continuously improve on those principles.
+
+## Culture
+
+The purpose of DevOps in the Apprenticeship service is to bring unity among the teams so that they all work as a single entity to achieve a goal or rollout new features and services. Working together creates a positive attitude and in turn improves the overall environment for the teams to work in. This enables continuous learning and sharing of skills. All this leads to efficient and innovative development and delivery. These are some of the cultural advantages DevOps provides for the service:
+
+- **Agile ways of working.** Across the service we perform daily collaboration through Agile ceremonies; sprint planning, daily standup, sprint reviews and retrospectives. Support and trust are key to our ways of working which helps enable our continuous delivery. There are also other ceremonies that bring all of the teams together such as the Scrum of Scrums, Clan Leads meeting, Architecture board, Quality Guild and a regular service wide standup.
+
+- **Show and tell.** These are regular meetings that allow teams to celebrate their work and talk about what they’ve learned. It's a chance to bring teams together, share success and improve on collaboration. Progress is highlighted along with question and feedback sessions. Transparency is encouraged to glimpse 'behind the curtain' of the products in development and to highlight where dependencies may exist between teams. A great show and tell session is one that is performed on a regular basis to ensure continuous feedback.
+
+## Automation
+
+This is the most vital factor of the DevOps culture. The purpose of automation is to build a system such that it eliminates various human errors and can fix errors and repeating the successful process again without any sort of intervention or delays. Automation greatly improves the workflow and productivity of the service. 
+
+- **Infrastructure as Code (IaC).** IaC is an infrastructure management approach that makes continuous delivery and DevOps possible. Without IaC, our developers and engineers would have to treat each target environment individually, which becomes a tedious task as we have different environments for development, testing, and production use. We include the Azure Resource Manager (ARM) templates and Azure DevOps build definitions along side each an applications repository in GitHub, this is to match the development lifecycle of an application. As application requirements change, so can the build and release along side it. 
```

Comment:
bit of grammar: `...along side each an applications repository...`

Reply:
Sorted 👍